### PR TITLE
Mappings for net/minecraft/world/level/chunk/storage

### PR DESCRIPTION
**Pull request by noeppi-noeppi**

### CODE REVIEW

File: data/net/minecraft/world/level/chunk/storage/EntityStorage.mapping [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/world/level/chunk/storage/EntityStorage
+	METHOD <init> (Lnet/minecraft/server/level/ServerLevel;Ljava/io/File;Lcom/mojang/datafixers/DataFixer;ZLjava/util/concurrent/Executor;)V
+		ARG 1 level
+		ARG 2 regionFolder
+		ARG 3 fixerUpper
+		ARG 4 sync
+		ARG 5 executor
```
Comment (sciwhiz12) [inline data/net/minecraft/world/level/chunk/storage/EntityStorage.mapping]: Something brought up in the mentioned PR is the (yet-to-be-fully-resolved) point of 'parameters follow field names': `executor` -> `mainThreadExecutor`.

File: data/net/minecraft/world/level/chunk/storage/RegionBitmap.mapping [hunk 2]
```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/world/level/chunk/storage/RegionBitmap net/minecraft/world/level/chunk/storage/RegionBitmap
+	COMMENT Keeps track of which parts of a region file are used and which parts are free.
+	METHOD allocate (I)I
+		COMMENT Gets a valid offset inside the region file with wnough space to store the given amount of sectors and marks tha tspace as used.
```
Comment (sciwhiz12) [inline data/net/minecraft/world/level/chunk/storage/RegionBitmap.mapping]: Typos:
```suggestion
		COMMENT Gets a valid offset inside the region file with enough space to store the given amount of sectors and marks that space as used.
```

File: data/net/minecraft/world/level/chunk/storage/RegionFile.mapping [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	METHOD write (Lnet/minecraft/world/level/ChunkPos;Ljava/nio/ByteBuffer;)V
+		ARG 1 chunkPos
+		ARG 2 chunkData
+	METHOD writeToExternalFile (Ljava/nio/file/Path;Ljava/nio/ByteBuffer;)Lnet/minecraft/world/level/chunk/storage/RegionFile$CommitOp;
+		COMMENT Writes a chunk to a separate file with only that chunk. This is used for chunks larger than 1 MiB
+		ARG 1 externalChunkFile
+		ARG 2 chunkData
+	CLASS ChunkBuffer
+		METHOD <init> (Lnet/minecraft/world/level/chunk/storage/RegionFile;Lnet/minecraft/world/level/ChunkPos;)V
+			ARG 2 chunkPos
```
Comment (sciwhiz12) [inline data/net/minecraft/world/level/chunk/storage/RegionFile.mapping]: Note the brought-up point of 'parameters follow field names': `chunkPos` -> `pos`.

File: data/net/minecraft/world/level/chunk/storage/RegionFileStorage.mapping [hunk 4]
```diff
@@ -1,8 +1,12 @@
-CLASS net/minecraft/world/level/chunk/storage/RegionFileStorage
+CLASS net/minecraft/world/level/chunk/storage/RegionFileStorage net/minecraft/world/level/chunk/storage/RegionFileStorage
+	COMMENT Handles reading and writing the {@link net.minecraft.world.level.chunk.storage.RegionFile region files} for a {@link net.minecraft.world.level.Level}.
+	METHOD <init> (Ljava/io/File;Z)V
+		ARG 1 regionFolder
```
Comment (sciwhiz12) [inline data/net/minecraft/world/level/chunk/storage/RegionFileStorage.mapping]: Note the brought-up point of 'parameters follow field names': `regionFolder` -> `folder`.

File: data/net/minecraft/world/level/chunk/storage/RegionFileVersion.mapping [hunk 5]
```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/world/level/chunk/storage/RegionFileVersion net/minecraft/world/level/chunk/storage/RegionFileVersion
+	COMMENT A decorator for input and output streams used to read and write the chunk data from region files..
```
Comment (sciwhiz12) [inline data/net/minecraft/world/level/chunk/storage/RegionFileVersion.mapping]: The comment should also describe that the class represents the known versions (compression methods) of the region file format, possibly.

**Review by sciwhiz12 — CHANGES_REQUESTED** [review]

**Comment by sciwhiz12** [comment]
This may conflict with changes in #19 (depending on whichever may be merged in first).

**Review by sciwhiz12 — APPROVED** [review]

**Review by jriwanek — APPROVED** [review]

**Pull request merged**